src/pages/auth/AuthPage.tsx

```diff
@@ -92,38 +110,45 @@ export const AuthPage = () => {
   }, [popup]);
 
   return (
-    <div className="h-screen w-screen bg-game">
-      <div className="flex h-full flex-col justify-evenly">
-        <div className="flex h-full flex-col items-center justify-center gap-4 text-3xl text-primary-deep-dark">
-          <Lottie lottieOptions={lottieOptions} className="max-w-80" />
-          <div>스낵 게임</div>
-        </div>
-        <div className="mx-auto my-12 flex w-5/6  flex-col items-center justify-end gap-4">
-          <div
-            className="mx-4 flex h-10 w-full max-w-52 items-center justify-center gap-4 rounded-xl bg-[#FEE500]"
-            onClick={() =>
-              openOAuthDialog({ url: PATH.KAKAO, name: 'Login with KAKAO' })
-            }
-          >
-            <KakakoIcon className="h-10 w-10" />
-            <span className="w-20  text-center text-sm">카카오 로그인</span>
-          </div>
-          <div
-            className="mx-4 flex h-10 w-full max-w-52 items-center justify-center gap-4 rounded-xl bg-white"
-            onClick={() =>
-              openOAuthDialog({ url: PATH.GOOGLE, name: 'Login with Google' })
-            }
-          >
-            <GoogleIcon className="h-10 w-10" />
-            <span className="w-20 text-center text-sm text-[#6B7280]">
-              구글 로그인
-            </span>
+    <>
+      <SnackRainContainer />
+      <div className="h-screen w-screen ">
+        <div className="flex h-full flex-col justify-evenly">
+          <div className="flex h-full flex-col items-center justify-evenly gap-4 text-3xl text-primary-deep-dark">
+            <img
+              className="w-1/2 "
+              ref={logoRef}
+              src={LogoImage}
+              alt="무한 모드"
```

Comment:
요기 alt text가 수정이 덜 된 거 같아요!
```suggestion
              alt="로고"
```

정도는 우떨까용 ㅎ

Reply:
오호 좋아요~